geometry_msgs/Point position
int32 id